src/pspm_cfg/pspm_cfg_import.m

```diff
@@ -176,6 +176,12 @@
 smi_stimulus_resolution.help      = {['The resolution of the stimulus window. This field is required' ...
                                       'to perform px to mm conversions for gaze channels']};
 
+delimiter           = cfg_entry;
+delimiter.name      = 'Delimiter';
+delimiter.tag       = 'delimiter';
+delimiter.strtype   = 's';
+delimiter.help      = {['The delimiter to be used for file reading, leave blank to ']};

```

Comment:
I think you forget to finish the sentence xD

Reply:
ha ha thanks